package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@wikia/ad-engine": "git+https://github.com/Wikia/ad-engine.git#v41.9.3",
     "@wikia/ember-fandom": "github:Wikia/ember-fandom#20adaf9b214b67c1046090b6caed91aec92c472a",
-    "@wikia/search-tracking": "github:Wikia/search-tracking#1.2.0",
```

Comment:
Use version tag after https://github.com/Wikia/search-tracking/pull/10 is released